src/components/DatePicker/components/Day/Day.tsx

```diff
@@ -74,7 +75,9 @@ export const Day = memo(function Day({
     (focused || selected || today || date === 1) && !disabled ? 0 : -1;
   const ariaLabel = [
     `${today ? i18n.translate('Polaris.DatePicker.today') : ''}`,
-    `${Month[day.getMonth()]} `,
+    `${i18n.translate(
```

Comment:
So uh, it turns out here we were just using the enum key name (which would always be english) instead of actually translating month name.

Reply:
Good catch!